pkg/cmd/server/start/master_args.go

```diff
@@ -209,6 +209,8 @@ func (args MasterArgs) BuildSerializeableMasterConfig() (*configapi.MasterConfig
 		DNSConfig: &configapi.DNSConfig{
 			BindAddress: dnsServingInfo.BindAddress,
 			BindNetwork: dnsServingInfo.BindNetwork,
+
+			AllowRecursiveQueries: true,
```

Comment:
I'm fine making the default true for new configs and all-in-one startups, but leave the old behavior intact at https://github.com/openshift/origin/pull/8474/files/38a403227faaab4ab9de73ecef63d4ce9846dbb6#diff-d59c94963a713e75e3b30c98b1dc40f2R134 (unspecified in config means false)